test/steps/context.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
```

Comment:
Is this file new? Why the date is on 2019? Just run `make vet` or `make test` that the headers will be added for you.

Reply:
Changed